src/error-index-generator/main.rs

```diff
@@ -73,7 +73,7 @@ r##"<!DOCTYPE html>
 
     try!(write!(&mut output_file, "<h1>Rust Compiler Error Index</h1>\n"));
 
-    for (err_code, info) in err_map.iter() {
+    for (err_code, info) in err_map {
```

Comment:
Yeah, I don't think it's a problem at all, just an interesting side-effect of looking at the diff.